src/content/docs/en/recipes/call-endpoints.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Call endpoints from the server
+description: Learn how to call endpoints from the server in Astro.
+i18nReady: true
+type: recipe
+---
+
+## Recipe
```

Comment:
```suggestion
Calling an endpoint in Astro allows you to... This recipe uses a server endpoint to ... (VERY SPECIFICALLY WHAT EXACTLY THIS EXAMPLE DOES)

## Prerequisites

- A project with [SSR](/en/guides/server-side-rendering/) (output: 'server') enabled

## Recipe
```
Just an example of an introductary paragraph and of a Prerequisites section. If there are no prerequisites, let's still include the section, and just say `- none`!

Note that this recipe should be something very specific, that someone can follow through entirely and add something new to their page. So, it shouldn't be "here's what adding an endpoint is like..." It should be: "Follow these steps and your project will now do this." So, we do have to tell them what exactly is going to happen. It's then on the reader to adjust as necessary for their purposes to do something that is useful to them.  :)